frontend/src/LandingPage.js

```diff
@@ -0,0 +1,255 @@
+import React from 'react';
+import './App.css';
+import { Client, Status } from "@googlemaps/google-maps-services-js";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { Link } from 'react-router-dom';
+import { createBrowserHistory } from "history";
+import SECRET_KEY from './config';
+import Map from './Itinerary';
+import Button from "react-bootstrap/Button";
+import Attractions from './Attractions';
+import ReactDOM from 'react-dom';
+import ImgKey from './configImg';
+
+function LandingPage() {
+    function createListElem(attraction){
+        const liElement = document.createElement("li");
+        let touristsview = '';
+        const keyImg = Object.values(ImgKey)[0];
+        touristsview +='<img width=300 height= 200 src='+'https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photoreference='+attraction.photos[0].photo_reference+'&key='+keyImg+'>' +
+                                    '<br>' +
+                                    '<strong id="rating">'+attraction.rating+'</strong>' +
+                                    '<span id="total_rating">'+ '('+ attraction.user_ratings_total+ ')' +'</span>' +
+                                    '<br>' +
+                                    '<span id="name">'+attraction.name+'</span>' +
+                                    '<br>';
```

Comment:
I have copied this part from Hasan' s code base to make it work with the rest of the code